Makefile

```diff
@@ -1,11 +1,11 @@
 .PHONY:	rpm clean
 
-VERSION ?= 3.4.14
+VERSION ?= 3.5.9
 BUILD_NUMBER ?= 1
-SOURCE = zookeeper-$(VERSION).tar.gz
+SOURCE = apache-zookeeper-$(VERSION)-bin.tar.gz
 TOPDIR = /tmp/zookeeper-rpm
 PWD = $(shell pwd)
-URL = $(shell curl -s https://www.apache.org/dyn/closer.cgi/zookeeper/zookeeper-$(VERSION)/zookeeper-$(VERSION).tar.gz?asjson=1 | python -c 'import sys,json; data=json.load(sys.stdin); print data["preferred"] + data["path_info"]')
+URL = https://dlcdn.apache.org/zookeeper/zookeeper-$(VERSION)/$(SOURCE)
```

Comment:
This is the URL I see from the website. I don't think the pick closest mirror stuff is necessary.